libopenage/util/fslike/directory.cpp

```diff
@@ -272,7 +273,7 @@ bool Directory::unlink(const Path::parts_t &parts) {
 	return std::remove(this->resolve(parts).c_str()) == 0;
 }
 
-
+// Need to research how std::filesystem::last_write_time plays with different platforms
```

Comment:
https://en.cppreference.com/w/cpp/filesystem/last_write_time

Reply:
It's mentioned here for `vs-2017` even so should be fine
https://docs.microsoft.com/en-us/cpp/standard-library/filesystem-functions?view=vs-2017#last_write_time